### PR TITLE
Tricky file

### PR DESCRIPTION
**Pull request by double-beep**
Add tricky file to master.

### CODE REVIEW

**Comment by double-beep** [comment]
Just for completing this pull request.

File: trick.bat [hunk 1]
```diff
@@ -0,0 +1,7 @@
+@echo off
+color 0a
+echo This is a trick!
+
+:TOP
+echo %random%
+goto top
```
Comment (helper-beep) [inline trick.bat]: Needed a lot.

**Comment by helper-beep** [comment]
ok.

**Comment by helper-beep** [comment]
ok.

**Pull request merged**